uint8 agentIdx
int8 TaskType
bool systemWorkingStatus
